azure-pipelines.yml

```diff
@@ -11,12 +11,32 @@ trigger:
     include:
       - '*'
 
+variables:
+  MAVEN_CACHE_FOLDER: $(Pipeline.Workspace)/.m2/repository
+  MAVEN_OPTS: '-Dmaven.repo.local=$(MAVEN_CACHE_FOLDER)'
+
 pool:
   vmImage: 'ubuntu-latest'
 
 container: 'maven:3.6-jdk-8-slim'
 
 steps:
+- task: CacheBeta@0
```

Comment:
Could you please add a link to the docs / GitHub issue for this beta task to the commit message?